bindings/node/etc/build-static.sh

```diff
@@ -23,22 +23,30 @@ pushd $BUILD_DIR #./deps/tmp
 pushd $TOP_DIR
 # build and install bson
 
-# NOTE: we are setting -DCMAKE_INSTALL_LIBDIR=lib to ensure that the built 
+# NOTE: we are setting -DCMAKE_INSTALL_LIBDIR=lib to ensure that the built
 # files are always installed to lib instead of alternate directories like
 # lib64.
 # NOTE: On OSX, -DCMAKE_OSX_DEPLOYMENT_TARGET can be set to an OSX version
 # to suppress build warnings. However, doing that tends to break some
 # of the versions that can be built
-export BSON_EXTRA_CMAKE_FLAGS="-DCMAKE_INSTALL_LIBDIR=lib -DCMAKE_OSX_DEPLOYMENT_TARGET=\"10.12\""
-
+export BSON_EXTRA_CMAKE_FLAGS="-DCMAKE_INSTALL_LIBDIR=lib -DCMAKE_OSX_DEPLOYMENT_TARGET=\"10.12\" -DCMAKE_C_FLAGS_RELWITHDEBINFO=\"/MT\""
```

Comment:
should this flag be here even for non-windows builds?

Reply:
it will be ignored for non-windows builds, similarly `-fPIC` is always added, but ignored on windows builds